src/main/resources/gamedata/species.yml

```diff
@@ -0,0 +1,34 @@
+################################################
+#                                              #
+#            YAML SPECIES DATABASE             #
+#                                              #
+#    This configuration file is tasked with    #
+#    declaring various species of different    #
```

Comment:
Of different what? You might have missed a word there.

Reply:
I got so involved in making sure the text is centered and pretty I forgot to specify the different whats indeed. They're supposed to be ants. So I wrote that in.